packages/fe-fpm-writer/test/integration/__snapshots__/index.test.ts.snap

```diff
@@ -403,6 +403,30 @@ Object {
     \\"archeType\\": \\"transactional\\"
   }
 }
+",
+    "state": "modified",
+  },
+  "ts/webapp/Component.ts": Object {
+    "contents": "import AppComponent from 'sap/fe/core/AppComponent';
+
+/**
+ * @namespace v4travel
+ */
+export default class Component extends AppComponent {
+
+	public static metadata = {
+		manifest: \\"json\\"
+	};
+
+    /**
+     * The component is initialized by UI5 automatically during the startup of the app and calls the init method once.
+     * @public
+     * @override
+     */
+	//public init() : void {
+    //    super.init();
+	//}
+}
```

Comment:
Can this be removed or uncommented?

Reply:
I am ok with either. It is not needed but I wanted to show how to extend `init` if needed.
@devinea what would you prefer - uncommenting or removing?

Reply:
Uncomment it for it this is needed frequently. Otherwise remove it. Thanks

Reply:
Removed with https://github.com/SAP/open-ux-tools/pull/645/commits/a38da63d1672e28bdece2aab52a86f957865ecdf